Website with documentation (#3039)
openvinotoolkit/cvat: site/content/en/docs/for-developers/xml_format.md

---
title: 'XML annotation format'
linkTitle: 'XML annotation format'
weight: 3
---

<!--lint disable heading-style-->

When you want to download annotations from Computer Vision Annotation Tool (CVAT)
you can choose one of several data formats. The document describes XML annotation format.
Each format has X.Y version (e.g. 1.0). In general the major version (X) is incremented when the data format has
incompatible changes and the minor version (Y) is incremented when the data format is slightly modified
(e.g. it has one or several extra fields inside meta information).
The document will describe all changes for all versions of XML annotation format.

## Version 1.1

> There are two different formats for annotation and interpolation modes at the moment.

Reviewer: ```suggestion
There are two different formats for images and video tasks at the moment.
```

> Both formats have a common part which is described below. From previous version `flipped` tag was added.

Reviewer: ```suggestion
The both formats have a common part which is described below. From the previous version `flipped` tag was added.
```

### Annotation

> Below you can find description of the data format for annotation mode. In this mode images are annotated.

Reviewer: ```suggestion
Below you can find description of the data format for images tasks.
```

On each image it is possible to have many different objects. Each object can have multiple attributes.

> If an annotation task has been created with `z_order` flag then each object will have `z_order` attribute which is used

Reviewer: ```suggestion
If an annotation task is created with `z_order` flag then each object will have `z_order` attribute which is used
```

### Interpolation

> Below you can find description of the data format for interpolation mode. In the mode frames are annotated.

Reviewer: ```suggestion
Below you can find description of the data format for video tasks.
```

## Version 1

> There are two different formats for annotation and interpolation modes at the moment.

Reviewer: ```suggestion
There are two different formats for images and video tasks at the moment.
```

### Annotation

> Below you can find description of the data format for annotation mode. In the mode images are annotated.

Reviewer: ```suggestion
Below you can find description of the data format for images tasks.

```

### Interpolation

> Below you can find description of the data format for interpolation mode. In this mode frames are annotated.

Reviewer: ```suggestion
Below you can find description of the data format for video tasks.
```
